common/lumens.js

```diff
@@ -86,6 +103,33 @@ export function distributionPartnershipProgram() {
   );
 }
 
+function sumRelevantAccounts(accountsObj) {
```

Comment:
created this little helper method to iterate over and sum an obj of accounts as all the programs I was dealing with had to do that